pyproject.toml

```diff
@@ -1,65 +1,57 @@
 [build-system]
-requires = [
-    "setuptools>=61.0.0",
-]
+requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "nomad-analysis"
-version = '0.1.0'
-description = "A standard NOMAD plugin for analysis."
+version = '1.0'
+description = "A NOMAD plugin for analysis of FAIR data."
 readme = "README.md"
 authors = [
-    { name = "Sarthak Kapoor", email = "sarthak.kapoor@physik.hu-berlin.de" }
+    { name = "Sarthak Kapoor", email = "sarthak.kapoor@physik.hu-berlin.de" },
+    { name = "Jose M. Pizarro", email = "jose.pizarro@physik.hu-berlin.de" },
+    { name = "Sherjeel Shabih", email = "sherjeel.shabih@physik.hu-berlin.de" }
 ]
 license = { text = "Apache-2.0" }
 requires-python = ">=3.9"
-classifiers = [
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.9",
-    "License :: OSI Approved :: Apache Software License",
-]
 dependencies = [
     "nomad-lab>=1.2.2dev399",
     "nbformat>=5.9.2",
 ]
+
+[project.urls]
+homepage = "https://github.com/FAIRmat-NFDI/nomad-analysis"
+
 [project.optional-dependencies]
 dev = [
     "ruff>=0.2.0",
     "pytest",
     "structlog==22.3.0"
 ]
 
-[project.urls]
-"Homepage" = "https://github.com/FAIRmat-NFDI/nomad-analysis"
-"Bug Tracker" = "https://github.com/FAIRmat-NFDI/nomad-analysis/issues"
```

Comment:
What is the reason for excluding the bug tracker?

Reply:
We can add as much as we want. Also I would suggest also to add "Documentation"

Reply:
Ok, I added them back, and also "Documentation". Check it out in lines 24-29